modules/administration-guide/pages/importing-untrusted-tls-certificates.adoc

```diff
@@ -61,58 +55,63 @@ Certificate files are typically stored as Base64 files, with extensions such as
 $ {orch-cli} create configmap custom-certs --from-file=__<bundle-file-path>__ -n={prod-namespace}
 ----
 +
-Add another `-from-file=_<bundle-file-path>_` flag to apply more than one bundle. Otherwise, create another ConfigMap.
+To apply more than one bundle, add another `-from-file=_<bundle-file-path>_`. Otherwise, create another ConfigMap. 
+// Wouldn't "alternatively" make more sense than "otherwise"? 
```

Comment:
You need to delete this peer review note/question: `// Wouldn't "alternatively" make more sense than "otherwise"?`